util/new_project.sh

```diff
@@ -43,7 +43,11 @@ GIT=$(whereis git)
 if [ "$GIT" != "" ]; then
   IS_GIT_REPO=$($GIT log >>/dev/null 2>&1; echo $?)
   if [ "$IS_GIT_REPO" -eq 0 ]; then
-    ID="'$($GIT config --get user.name)'"
+    ID="$($GIT config --get user.name)"
+    read -p "What is your name? [$ID] " YOUR_NAME
```

Comment:
```suggestion
    read -rp "What is your name? [$ID] " YOUR_NAME
```

should put a -r there in the strange event that someone wants to use backslashes for their name lol.